An error occurred while executing the following cell:
------------------
pml.latexify(width_scale_factor=2)
markersize = 3 if pml.is_latexify_enabled() else 6
------------------

---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
/tmp/ipykernel_4776/355754833.py in <module>
----> 1 pml.latexify(width_scale_factor=2)
      2 markersize = 3 if pml.is_latexify_enabled() else 6

NameError: name 'pml' is not defined
NameError: name 'pml' is not defined
